lib/graphql/introspection/directive_type.rb

```diff
@@ -4,7 +4,5 @@
   field :name, !types.String, "The name of this directive"
   field :description, types.String, "The description for this type"
   field :args, field: GraphQL::Introspection::ArgumentsField
-  field :onOperation, !types.Boolean, "Does this directive apply to operations?", property: :on_operation?
```

Comment:
Oops, error on my part! They can be re-implemented & marked as deprecated, sorry!